music21/musicxml/partStaffExporter.py

```diff
@@ -705,11 +706,48 @@ def moveMeasureContents(measure: Element, otherMeasure: Element, staffNumber: in
             if elem.tag == 'print':
                 continue
             if elem.tag == 'attributes':
-                if elem.findall('divisions'):
-                    # This is likely the initial mxAttributes
-                    continue
                 for midMeasureClef in elem.findall('clef'):
                     midMeasureClef.set('number', str(staffNumber))
+
+                # Determine how much of the <attributes> tag to move
+                target_measure_meters_or_none = otherMeasure.findall('attributes/time')
+                target_measure_keys_or_none = otherMeasure.findall('attributes/key')
+                source_measure_meter_or_none = elem.find('time')
+                source_measure_key_or_none = elem.find('key')
+                skippable_meter = False
+                skippable_key = False
```

Comment:
this works, but I don't like how much everything is moving into musicxml/Element manipulation.  I had qualms about it before when the partStaffExporter started doing it, but I want to stop before it becomes a norm in the musicxml manipulation.  We should identify skippable and identical music21 elements and remove them from the stream before outputting, not remove things from the XML DOM.